data/themes/darktable.css

```diff
@@ -920,6 +921,7 @@ checkbutton check
 }
 
 #left #history-button, /* choose here spacing between history items in darkroom */
+#left #history-button-enabled,
 #left #history-switch,
 #left #history-button-always-enabled,
 #left #history-switch-always-enabled,
```

Comment:
You forgot to add deprecated switch here. You just have to add :
`#left #history-switch-deprecated`